src/CostManagement/custom/New-AzCostManagementQueryColumnObject.ps1

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
Looks this client side cmdlet is not required.

Reply:
Yes.This cmdlet will be deleted in subsequent submissions.